vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,138 +4,145 @@ sidebarDepth: 2
 # Build and Publish a Simple Bundle
 
 ## Overview
-In this tutorial, you will learn how to create a simple Entando Bundle and deploy it into the [Entando Component Repository](../../../docs/getting-started/concepts-overview.md#entando-component-repository) (ECR). This involves manually defining a bundle with a single widget, checking the bundle artifacts into Git, applying the Entando Bundle custom resource to Kubernetes, and then installing the bundle into an application.
+
+This tutorial describes how to create a simple Entando Bundle and deploy it into the [Entando Component Repository](../../../docs/getting-started/concepts-overview.md#entando-component-repository) (ECR). This process consists of 4 essential steps:
+
+1. Define the bundle component(s)
+2. Check the bundle artifacts into a Git repository
```

Comment:
This step is incorrect for 7.1. Change to "Publish the bundle artifacts to a Docker registry."